chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -61,6 +61,25 @@
   include_recipe "ceph::radosgw_keystone"
 end
 
+# Create admin user and add caps to access buckets and users
+ruby_block "create Radosgw-admin-access" do
+  block do
+    uid = "rgw_admin"
+
+    user_add = Mixlib::ShellOut.new("radosgw-admin user create --display-name=RadosGW_Admin --uid=#{uid}")
```

Comment:
Metrics/LineLength: Line is too long. [106/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)